packages/rancher-monitoring/rancher-windows-exporter/charts/values.yaml

```diff
@@ -19,15 +18,22 @@ serviceMonitor:
 prometheusRule:
   enabled: true
 
+# Security context configuration for privileged HostProcess Windows pods
+securityContext:
+  windowsOptions:
+    hostProcess: true
+    runAsUserName: "NT AUTHORITY\\SYSTEM"
+
 ## Components scraping metrics from Windows nodes
 ##
 clients:
   enabled: true
 
+  maxRequests: 5
   port: 9796
   image:
     repository: rancher/windows_exporter-package
-    tag: v0.0.3
```

Comment:
Note: This image isn't final, we will likely cut `v0.1.0` or directly use the upstream image - tbd